src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Lottery {
+    private static ColorSupplier colorSupplier = new ColorSupplier();
+
+    public static Ball getRandomBall() {
```

Comment:
let's make this method non-static
and call it through object of Lottery class in main()